src/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```

Comment:
To logo zostalo usuniete, poniewaz nie bedzie nam potrzebne. To jest logo Reacta wykorzystane na stronie powitalnej po stworzeniu projektu z react-create-app